packages/staging-config/src/baseUrl.ts

```diff
@@ -0,0 +1,127 @@
+// mimicking values from https://github.com/geoadmin/web-mapviewer/blob/36043456b820b03f380804a63e2cac1a8a1850bc/packages/mapviewer/src/config/staging.config.js#L1-L7
+export type Staging = 'development' | 'integration' | 'production'
+
+export type BackendServices = 'wms' | 'wmts' | 'api3' | 'data' | 'kml' | 'shortlink' | 'tiles3D' | 'vectorTiles' | 'proxy' | 'viewerSpecific'
+
+export type ServiceBaseUrl = {
+    [key in Staging]: string
+}
+export type ServicesBaseUrl = {
+    [key in BackendServices]: ServiceBaseUrl
+}
+
+export const servicesBaseUrl: ServicesBaseUrl = {
+    /**
+     * Default WMS base part of the URL to use when requesting tiles (e.g. for prod
+     * https://wms.geo.admin.ch).
+     *
+     * WMS layers tend to carry their own base URL in their metadata, be mindful of that when using
+     * this constant (it might be unnecessary)
+     *
+     * @see https://github.com/geoadmin/service-wms
+     */
+    'wms': {
+        'development': 'https://sys-wms.dev.bgdi.ch/',
+        'integration': 'https://sys-wms.int.bgdi.ch/',
+        'production': 'https://wms.geo.admin.ch/'
+    },

```

Comment:
why use strings for the attributes? why not just like this?
```suggestion
    wms: {
        development: 'https://sys-wms.dev.bgdi.ch/',
        integration: 'https://sys-wms.int.bgdi.ch/',
        production: 'https://wms.geo.admin.ch/'
    },

```